.github/workflows/test-pr.yml

```diff
@@ -155,6 +155,37 @@ jobs:
         run: |
           docker stop --time=0 kontrol-ci-integration-${GITHUB_SHA}
 
+  end-to-end-tests:
+    needs: code-quality-checks
+    name: 'End to End Sanity Tests'
+    runs-on: [self-hosted, linux, normal, fast]
+    strategy:
+      fail-fast: false
+      matrix:
+        backend: ['booster']
```

Comment:
```suggestion
```

It looks like this matrix isn't used at all, we just run with the same `TEST_ARGS` either way (only with the booster).

Reply:
Addressed in https://github.com/runtimeverification/kontrol/pull/507/commits/3053c358d30f720fb1b8beddd9519e55fafc12e4.